libs/langchain/langchain/vectorstores/tiledb.py

```diff
@@ -359,16 +414,77 @@ def max_marginal_relevance_search(
         )
         return docs
 
+    @classmethod
+    def create(
+        cls,
+        index_uri: str,
+        index_type: str,
+        dimensions: int,
+        vector_type: np.dtype,
+        metadatas: bool = True,
+        config: Optional[Mapping[str, Any]] = None,
+    ) -> None:
+        tiledb_vs, tiledb = dependable_tiledb_import()
+        with tiledb.scope_ctx(ctx_or_config=config):
+            try:
+                tiledb.group_create(index_uri)
+            except tiledb.TileDBError as err:
+                raise err
+            group = tiledb.Group(index_uri, "w")
+            vector_index_uri = f"{group.uri}/{VECTOR_INDEX_NAME}"
```

Comment:
These URI constructions are repeated quite a bit, considering moving to a helper

Reply:
Done